packages/uikit/src/util/withTypography.ts

```diff
@@ -0,0 +1,35 @@
+import { css } from '../lib/styledComponents';
+
+interface TypographyProps {
+  align: 'center' | 'left' | 'right';
+  underline: boolean;
+  lineHeight: number;
+  fontWeight: 'lighter' | 'regular' | 'bold';
+}
+
+export default ({
+  align,
+  underline,
+  lineHeight,
+  fontWeight
+}: TypographyProps) => css`
```

Comment:
all attributes apart from lineHeight should probably either have a default value or not be required I think.